src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,50 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
-        List<Product> products = getAllProductsFromShoppingCart(userId);
+        logger.info("Starting the process of completing an order for userId: {}", userId);
+        List<Product> products = null;
+        logger.info("Retrieving products from the shopping cart");
+        try {
+            products = getAllProductsFromShoppingCart(userId);
+            logger.debug("Products retrieved from shopping cart for userId: {}: {}",
+                    userId, products);
+        } catch (Exception e) {
+            logger.error("Failed to retrieve products from shopping cart for userId: {}",
+                    userId, e);
+            throw new RuntimeException("Please try again");
```

Comment:
Throwing a generic RuntimeException with a generic message is not informative. Consider creating a custom exception that can provide more context about the error.